src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -362,6 +411,9 @@ pub fn add_new_sub_level(
     insert_sub_level_id: u64,
     insert_table_infos: Vec<SstableInfo>,
 ) {
+    if insert_sub_level_id == u64::MAX {
```

Comment:
When will this be true?

Reply:
> When will this be true?

When summary doesn't find any insert operations in `level_delta`